osu.Game/Overlays/BeatmapListing/FilterTabItem.cs

```diff
@@ -44,7 +44,14 @@ private void load()
             });
 
             Enabled.Value = true;
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
             updateState();
+            FinishTransforms(true);
         }
```

Comment:
This is required due to BDL ordering foibles; the base BDL from `FilterTabItem` was calling `updateState()`, which was calling `GetStateColour()` in `FeaturedArtistsTabItem`, making it impossible to access locally-resolved `OsuColour` from there.

The `FinishTransforms(true)` is a bonus because `updateState()` actually adds a transform with duration:

https://github.com/ppy/osu/blob/65ffcefdf9ec2e136b426378d986dd629cbd28a6/osu.Game/Overlays/BeatmapListing/FilterTabItem.cs#L74

Not sure how that never came up as an issue. May have been saved by early loads, or the wave transition when the beatmap listing overlay is opened.